docs/conf.py

```diff
@@ -83,7 +83,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'F5 Marathon BIG-IP Controller'
+project = u'F5 BIG-IP Controller for Marathon'
 copyright = u'2016,2017 F5 Networks Inc'
```

Comment:
I think we should list all years that the product has existed in the copyright. Also, it is a separate change from the rename effort.